src/components/sbb-clock/sbb-clock.spec.ts

```diff
@@ -1,24 +1,25 @@
-import { SbbClock } from './sbb-clock';
-import { newSpecPage } from '@stencil/core/testing';
+import { expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import './sbb-clock';
 
 describe('sbb-clock', () => {
   it('renders', async () => {
-    const { root } = await newSpecPage({
-      components: [SbbClock],
-      html: '<sbb-clock />',
-    });
+    const root = await fixture(html`<sbb-clock />`);
```

Comment:
In this specific case you won't have a problem, but better be consistent imho
```suggestion
    const root = await fixture(html`<sbb-clock></sbb-block>`);
```